### PR TITLE
Docs: remove MSYS requirement for Windows

### PR DESCRIPTION
**Pull request by laszlocsomor**
See https://github.com/bazelbuild/bazel/issues/10003

Change-Id: I573000c150bd7a273ec0d2ff3389e774677cb6d5

### CODE REVIEW

File: site/docs/install-windows.md [hunk 1]
```diff
@@ -42,16 +33,67 @@ Recommended: rename this binary to `bazel.exe` and move it to a directory on the
...
+## 4. Optional: configure output directories
+
+**You can skip this step. Bazel can work without configuring the output directories, and will use
+its default values.**
+
+By default, Bazel writes to two directories:
+
+-   The "output user root", configurable with the `--output_user_root` flag.
+
+    This is where Bazel extracts from itself some embedded tools, its own Java runtime, and where it
```
Comment (meteorcloudy) [inline site/docs/install-windows.md]: Nit: Hmm, do we need to mention the Java runtime? Some people may not like this, although it's not a really issue for them.

File: site/docs/install-windows.md [hunk 2]
```diff
@@ -68,7 +110,7 @@ We recommend installing:
 
     You will need this to build Java code on Windows.
 
-    Also supported: Java 8, 9
+    Also supported: Java 8 and 9
 
 *   [Python 2.7 for Windows x86-64](https://www.python.org/downloads/windows/)
```
Comment (meteorcloudy) [inline site/docs/install-windows.md]: Can we change the default to Python 3?

Comment (laszlocsomor) [reply site/docs/install-windows.md]: I'm wary of recommending py3 unless we know it works. Py2 is deprecated but it works.

Which version do we use on CI? If it's py3, that'd give me confidence that we can recommend it.

File: site/docs/install-windows.md (hunk 1 above)
Comment (laszlocsomor) [reply site/docs/install-windows.md]: Good idea. I'm not happy about omitting it (why should we hide it?), but I know it has been controversial (for questionable reasons). Removing.

File: site/docs/install-windows.md (hunk 2 above)
Comment (meteorcloudy) [reply site/docs/install-windows.md]: We installed both versions, but use python3 as default on Windows machines. I'm also using Python3 for everyday develop, so it should be fine. But we can also do it after the python 3 migration we're going to do.

Comment (laszlocsomor) [reply site/docs/install-windows.md]: SGTM, let's defer updating this section.

**Pull request closed without merging**